calico-cloud/get-started/connect/usage-metrics.mdx

```diff
@@ -11,4 +11,11 @@ In Manager UI:
 1. From the cluster drop-down menu, select your cluster.
 1. Click the Profile icon, and select **Usage Metrics**.
 
-![usage-metrics](/img/calico-cloud/usage-metrics.png)
\ No newline at end of file
+![usage-metrics](/img/calico-cloud/usage-metrics.png)
+
+## How Node Hours are calculated
+
+`Node Hours` are calculated for a single node by dividing the vCPU of each node by four, rounding up, and then *adding them together*. Because the calculation is on a per-node basis, the results are not the same as adding up all the vCPU and dividing by four. `Node Hours` are updated every hour, and change every month because of the different number of days in a month.
```

Comment:
I find this first sentence confusing as it jumps between "a single node" and "each node" in the same sentence

>Node Hours are calculated for **a single node** by dividing the vCPU **of each node** by four